index.html

```diff
@@ -1,19 +1,93 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
+	<meta charset="utf-8">
 
-  <title>My Resume</title>
+	<title>My Resume</title>
 
-  <link rel="stylesheet" href="css/index.css">
+	<link rel="stylesheet" href="css/style.css">
+	<link href="https://fonts.googleapis.com/css?family=Roboto:400,500" rel="stylesheet">
```

Comment:
:heavy_check_mark:

Reply:
Thanks for the feedback. I did correct the work history table.

Do you prefer using LESS vs. vanilla CSS?
* I do feel comfortable with LESS. I think it's easier than vanilla but i can see a use for both.

Do you feel comfortable with all of the review objectives?
* Yes